bubblesub/ui/mpv.py

```diff
@@ -14,6 +14,8 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+# pylint: disable=no-member
```

Comment:
Are there so many warnings that this needs to be on module level?

Reply:
Lots of [them](https://github.com/Luni-4/bubblesub/commit/cee53fb666174476a9041f7a3785cc1492634cbf/checks?check_suite_id=383712139#step:10:8)